drivers/bluetooth/btrtl.c

```diff
@@ -200,6 +219,17 @@ static int rtl8723b_parse_firmware(struct hci_dev *hdev, u16 lmp_subver,
 		return -ENOMEM;
 
 	memcpy(buf + patch_length - 4, &epatch_info->fw_version, 4);
+	
+	/* Workaround for Foxconn MT13. 
+	 * The default BT antenna port for this laptop is not populated,
+	 * so we force Antenna S0.
+	 */
+	if (dmi_check_system(bt_force_S0_antenna)) {
+		BT_INFO("USING_S0_ANTENNA");
```

Comment:
You probably want BT_DBG here, since this message is not really meaningful to the regular user. It is probably a good idea to use spaces instead of underscore there, and avoid all caps.